src/loader-components/_ExampleTemplate.jsx

```diff
@@ -33,12 +38,107 @@ const defaultProps = {
   isExpanded: false,
 };
 
+/**
+ * Enables focus styles for the target of the given event. Typically used as an onBlur callback on selectable elements.
+ */
+const enableFocusStyles = (event) => {
+  event.currentTarget.setAttribute('data-focus-styles-enabled', 'true');
+};
+
+/**
+ * Disables focus styles for the target of the given event. Typically used as an onMouseDown callback on selectable elements.
+ */
+const disableFocusStyles = (event) => {
+  event.currentTarget.setAttribute('data-focus-styles-enabled', 'false');
+};
+
 const ExampleTemplate = ({
-  example, exampleSrc, title, description, isExpanded,
+  example, exampleSrc, exampleCssSrc, title, description, isExpanded,
 }) => {
   const [codeIsVisible, setCodeIsVisible] = useState(isExpanded);
-  const [isBackgroundTransparent, setIsBackgroundTransparent] = useState(false);
+  const [cssIsVisible, setCssIsVisible] = useState(false);
   const theme = React.useContext(ThemeContext);
+  const cssButtonVisible = exampleCssSrc !== undefined;
+
+  const handleCssToggle = () => {
+    setCssIsVisible(!cssIsVisible);
+    if (codeIsVisible) {
+      setCodeIsVisible(!codeIsVisible);
+    }
+  };
+
+  const handleCodeToggle = () => {
+    setCodeIsVisible(!codeIsVisible);
+
+    if (cssIsVisible) {
+      setCssIsVisible(!cssIsVisible);
+    }
+  };
+
+  const handleKeyDown = (event, handleToggle) => {
+    if (event.nativeEvent.keyCode === KeyCode.KEY_SPACE || event.nativeEvent.keyCode === KeyCode.KEY_RETURN) {
+      event.preventDefault();
+      handleToggle();
+    }
+  };
+
+  const renderExamples = () => (
+    <div>
+      { cssIsVisible
+        && (
+          <div className={cx('css')}>
+            {exampleCssSrc}
+          </div>
+        )}
+      { codeIsVisible
+        && (
+          <div className={cx('code')}>
+            {exampleSrc}
+          </div>
+        )}
+    </div>
+  );
+
+  const renderButtons = () => (
+    <div>
+      {exampleSrc
+      && (
+      <div className={cx('footer')}>
+        <div className={cx('button-container')}>
+          {cssButtonVisible && (
+            <button
+              type="button"
+              className={cx('css-toggle', 'item', { 'is-selected': cssIsVisible })}
+              onClick={handleCssToggle}
+              onKeyDown={event => handleKeyDown(event, handleCssToggle)}
+              onBlur={enableFocusStyles}
+              onMouseDown={disableFocusStyles}
+              tabIndex={0}
+              data-focus-styles-enabled
+            >
+              CSS
```

Comment:
can we add in the < > around the css like the code button does?

Reply:
Added chevrons and updated all the wdio snapshots: [4984d2d](https://github.com/cerner/terra-dev-site/pull/281/commits/4984d2d10c2e10137cfbca58cfbd747cce3e5d83).